.github/workflows/develop.yml

```diff
@@ -720,6 +719,9 @@ jobs:
           save-if: ${{ github.ref == 'refs/heads/main' }}
       - name: Install latest nextest release
         uses: taiki-e/install-action@nextest
+      - name: Setup postgres
```

Comment:
The Postgres service is started in [c651bb8](https://github.com/GreptimeTeam/greptimedb/pull/6764/commits/c651bb87bcf47b10e0ac5c45d9269523f9465f35), so we can remove this debug step and let all containers start in the background.